grid-proxy/pkg/types/nodes.go

```diff
@@ -48,6 +48,7 @@ type Node struct {
 	ExtraFee          uint64       `json:"extraFee" sort:"extra_fee"`
 	Healthy           bool         `json:"healthy"`
 	PriceUsd          float64      `json:"price_usd" sort:"price_usd"`
+	_                 string       `sort:"free_cru"`
```

Comment:
I am curious as what that does

Reply:
What does it do? :D

Reply:
- the validation on the `sort_by` params is done based on the fields that have a `sort` tag on the `Node` struct.
- but adding a `FreeCru` field to the Node struct will mostly show negative values due to the over-provisioning which may be misleading 

 doing it this way will provide `free_cru` as a valid sorting param without `FreeCru` as a Node field